components/eventing-controller/pkg/deployment/publisher_deployment.go

```diff
@@ -330,10 +330,10 @@ func getNATSEnvVars(natsConfig env.NatsConfig, publisherConfig env.PublisherConf
 				},
 			},
 		},
-		// jetstream-specific config
+		// Jetstream-specific config
 		{Name: "ENABLE_JETSTREAM_BACKEND", Value: strconv.FormatBool(natsConfig.EnableJetStreamBackend)},
 		{Name: "JS_STREAM_NAME", Value: natsConfig.JSStreamName},
-		{Name: "JS_STREAM_SUBJECT_PREFIX", Value: natsConfig.JSStreamSubjectPrefix},
+		{Name: "JS_STREAM_SUBJECT_PREFIX", Value: env.JetstreamSubjectPrefix},
```

Comment:
imho this setting does not need to be modified. we could just hardcode it everywhere. No real need for a environment variable

Reply:
done